index.md

```diff
@@ -132,9 +76,6 @@ set of quality reports.
 ### `QualityReport.wdl`
 A number of quality reports.
 
-### `ValidateFastqFiles.wdl`
-This workflow doesn't produce any output, but fails if validation fails.
-
 ## About
```

Comment:
This header should be removed and the contents of this section should be moved to above usage.